### ISSUE TITLE
Symphonia fails to play specific MP3

### ISSUE THREAD

**Issue by n0samu** [pdeljanov/Symphonia#190]
My apologies if this issue is a bit less detailed than it should be. I am a user of [Ruffle](https://github.com/ruffle-rs/ruffle/) trying to help investigate an issue (https://github.com/ruffle-rs/ruffle/issues/9676) with playing a specific MP3 sound. Ruffle uses Symphonia to play MP3 sounds, and I found that `symphonia_play` is unable to play the MP3, so I think that this may be a problem with Symphonia.

Here is the MP3 that Symphonia is unable to play: [12_BGM_01.zip](https://github.com/pdeljanov/Symphonia/files/10813486/12_BGM_01.zip)

The error output from `symphonia_play` is:
```
WARN  symphonia_bundle_mp3::demuxer > invalid mpeg audio header
WARN  symphonia_bundle_mp3::demuxer > skipping junk at 357 bytes
... (many more similar messages) ...
INFO  symphonia_play                > the input is not supported
ERROR symphonia_play                > end of stream
```

**Comment by pdeljanov** [comment]
Thanks for the report. This is now fixed with c7d7968.

**Comment by n0samu** [comment]
Great, thank you!
